vm-setup/roles/v1a2_integration_test/templates/controlplane_ubuntu.yaml

```diff
@@ -0,0 +1,108 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
+kind: BareMetalMachine
+metadata:
+  name: {{ CLUSTER_NAME }}-controlplane-0
+spec:
+  image:
+    url: {{ IMAGE_URL }}
+    checksum: {{ IMAGE_CHECKSUM }}
+---
+apiVersion: bootstrap.cluster.x-k8s.io/v1alpha2
+kind: KubeadmConfig
+metadata:
+  name: {{ CLUSTER_NAME }}-controlplane-0
+spec:
+  initConfiguration:
+    nodeRegistration:
+      name: {{ " '{{ ds.meta_data.name }}' " }}
+      kubeletExtraArgs:
+        node-labels: 'metal3.io/uuid={{ '{{ ds.meta_data.uuid }}' }}'
+  preKubeadmCommands:
+    - ip link set dev enp2s0 up
+    - dhclient enp2s0
+    - mv /tmp/akeys /home/ubuntu/.ssh/authorized_keys
+    - chown ubuntu:ubuntu /home/ubuntu/.ssh/authorized_keys
+    - apt update -y
+    - >-
+      apt install net-tools gcc linux-headers-$(uname -r) keepalived
+      apt-transport-https ca-certificates curl gnupg-agent
+      software-properties-common -y
+    - curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
+    - add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+    - curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+    - echo 'deb https://apt.kubernetes.io/ kubernetes-xenial main' > /etc/apt/sources.list.d/kubernetes.list
+    - apt update -y
+    - apt install docker-ce docker-ce-cli containerd.io kubelet kubeadm kubectl -y
+    - systemctl enable --now docker keepalived kubelet
+    - usermod -aG docker ubuntu
+  postKubeadmCommands:
+    - mkdir -p /home/ubuntu/.kube
+    - cp /etc/kubernetes/admin.conf /home/ubuntu/.kube/config
+    - chown ubuntu:ubuntu /home/ubuntu/.kube/config
+  files:
+      - path: /etc/keepalived/keepalived.conf
+        content: |
+          ! Configuration File for keepalived
+          global_defs {
+              notification_email {
+              sysadmin@example.com
+              support@example.com
+              }
+              notification_email_from lb@example.com
+              smtp_server localhost
+              smtp_connect_timeout 30
+          }
+          vrrp_instance VI_1 {
+              state MASTER
+              interface enp2s0
+              virtual_router_id 1
+              priority 101
+              advert_int 1
+              virtual_ipaddress {
+                  192.168.111.249
+              }#!/bin/bash
+apt update -y
+apt install net-tools gcc linux-headers-$(uname -r) curl \
+      apt-transport-https ca-certificates curl gnupg-agent \
+      software-properties-common keepalived -y
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
+add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+echo 'deb https://apt.kubernetes.io/ kubernetes-xenial main' > /etc/apt/sources.list.d/kubernetes.list
+apt update -y
+apt install docker-ce docker-ce-cli containerd.io \
```

Comment:
what is this ? unexpected copy paste, no ?

Reply:
It was the from the old commit, now it is fixed.